### ISSUE TITLE
Setting for transfer learning

### ISSUE THREAD

**Issue by th0214** [microsoft/RegionCLIP#43]
Hi, I have a question.
In train_transfer_learning.sh, I used RN50X4, COCO. But I had 4 gpu, so I changed num-gpus to 4 and changed IMS_PER_BATCH to 4. Then, the score written in the paper does not come out. In this case, what part should I change to get the score written in the paper?

**Comment by YiwuZhong** [comment]
@th0214 What score did you get after finetuning? If you decrease batch size to half, then you need to decrease learning rate to half accordingly.

**Comment by th0214** [comment]
I got a score like 
[11/24 14:36:18] d2.evaluation.coco_evaluation INFO: AP50_split_target AP: 0.3522030598755359
[11/24 14:36:18] d2.evaluation.coco_evaluation INFO: AP50_split_base AP: 0.5863819732395625
[11/24 14:36:18] d2.evaluation.coco_evaluation INFO: AP50_split_all AP: 0.5251351805135863

**Comment by YiwuZhong** [comment]
@th0214 Did you follow the suggestions above to re-train the model? The maximum iteration & steps when learning rate decreases are important in your case.
